docs/en/sql-reference/functions/string-functions.md

```diff
@@ -571,15 +571,13 @@ SELECT upper('clickhouse');
 
 Converts a string to lowercase, assuming that the string contains valid UTF-8 encoded text. If this assumption is violated, no exception is thrown and the result is undefined.
 
-Does not detect the language, e.g. for Turkish the result might not be exactly correct (i/İ vs. i/I).
```

Comment:
Why is it removed?

Reply:
Leftover of https://github.com/ClickHouse/ClickHouse/pull/65761